spec/imsc-hrm.html

```diff
@@ -229,9 +229,9 @@ <h2>Terms and Definitions</h2>
 
       <p><dfn>Related Media Object</dfn>. See Section 2.2 at [[!ttml2]].</p>
 
-      <p><dfn>presented region</dfn>. See [[!imsc]].</p>
+      <p><dfn>presented region</dfn>. See Section 8.12.1.1 at [[!imsc]].</p>
```

Comment:
I would rather remove the reference to specific sections, so that we do not need to maintain them across revisions of the references document.

Reply:
in any case, to remove or to add, I'd propose to align into one way..

or, how about to add a link to like: https://www.w3.org/TR/ttml-imsc/#dfn-presented-region